scripts/ci.ps1

```diff
@@ -46,7 +46,7 @@ Invoke-ChronicCommand "Compiling TypeScript" $yarn run compile-production
 
 # Get the CMake binary that we will use to run our tests
 # The cmake server mode has been removed since CMake 3.20. Clients should use the cmake-file-api(7) instead.
-$cmake_binary = Install-TestCMake -Version "3.18.2"
+$cmake_binary = Install-TestCMake -Version "3.18.6"
```

Comment:
Script is not being used, you can actually just delete the file.

Reply:
yes, both files should be deleted.